torch/onnx/symbolic_opset9.py

```diff
@@ -470,6 +470,12 @@ def expand(g, self, size, implicit):
 
 
 def expand_as(g, self, other):
+    self_t = sym_help._maybe_get_const(self, "t")
+    if isinstance(self_t, torch.Tensor):
+        self_t = self_t.to(torch.double)
+        if torch.equal(self_t, torch.full_like(self_t, torch.mean(self_t))):
```

Comment:
can this support non-scalar cases? e.g. `x[:, x.size(0):, :] = torch.tensor([1,2,3])` where `x.shape == [2, 5, 3]`.

Reply:
We are only supporting cases where right hand side is a constant tensor with uniform value.

Reply:
Do you mean the above is not supported, or it will not fall under this code path?

Reply:
So, I was thinking to keep it out of the scope, but it should be fixed now.